src/app/execution/txLogger.js

```diff
@@ -240,7 +236,7 @@ function renderKnownTransaction (self, data) {
         ${checkTxStatus(data.receipt, txType)}
         ${context(self, {from, to, data})}
         <div class=${css.buttons}>
-          <div class=${css.debug} onclick=${(e) => debug(e, data, self)}>Debug</div>
+          <button class="${css.debug} btn" onclick=${(e) => debug(e, data, self)}>Debug</div>
```

Comment:
The btn class isn't enough - you need to add 
btn btn-primary btn-sm  and also the debug class has too many definitions- it needs to have all the definitions taken out or it will interfere with the button sizing and font-size.